lib/new_relic/agent/llm_event/chat_completion.rb

```diff
@@ -0,0 +1,28 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+module NewRelic
+  module Agent
+    class LlmEvent
+      class ChatCompletion < LlmEvent
+        # TODO: should any of the attrs be required on initialization?
```

Comment:
since there are so many attributes, i'm down for not having any be required immediately, and just letting things be settable, so we could create the object first and then add attributes to it as we go? Maybe adding a []= method or something would be nice so we could just be like `event[:conversation_id] = 2`, or something like that? I think something like that would make the creation of all the attributes a little nicer to do than just having to provide everything in the initializer.
idk what does everyone think about something like that?